src/build-template.js

```diff
@@ -2,9 +2,19 @@ const fs = require("fs-extra");
 const Handlebars = require("handlebars");
 const moment = require("moment");
 
-module.exports.renderTemplate = function (changelogTemplate, data) {
+function injectMetadata(compiledTemplate, version) {
+  return `
+[//]: # (start ${version})
```

Comment:
What do you think about simplifying this metadata?

```
[//]: # s-${version}
[//]: # e-${version}
```

Reply:
In order to the comment properly work, we need to maintain the `()`, but I'll update to `s-` and `e-` instead of `start ` and `end `.

Reply:
Done :)